environment.yml

```diff
@@ -16,6 +16,12 @@ dependencies:
   - bioconda::pyprophet=2.1.4
   - conda-forge::networkx=2.4
   - bioconda::easypqp=0.1.8
+  - conda-forge::r-rmsnumpress=1.0.1
```

Comment:
Should the versions be ` >= ` instead of `=` ?

Reply:
I think it is better in this case to fix it to the specific version.

Reply:
Definitely, as we want the pipeline to use exactly the same versions if running in the future for reproducibility. In fact I think that the nf-core linting will fail if you don't use `=`.